.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
+.DS_Store
+public/dist
```

Comment:
You probably want to add `article-tagging-prototype/dist` here too, as you're currently pushing compiled files

Reply:
@jakemulley that was actually a folder that shouldn't have been in the repo; apologies. It dated from before the repo was to host multiple UI extensions. Have discarded and re-pushed.